doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-list-map-add-remove-item-proc.adoc

```diff
@@ -1,19 +1,55 @@
 [id='test-designer-list-map-add-remove-item-proc']
 = Using list and map collections in test scenarios
 
-The test scenarios designer supports list and map collections for both DMN-based as well as rules-based test scenarios. You can define a collection like a list or a map ​as the value of a particular cell in both *GIVEN* and *EXPECT* columns.
+The test scenarios designer supports list and map collections for both DMN-based as well as rules-based test scenarios. You can create and define a collection like a list or a map ​as the value of a particular cell in both *GIVEN* and *EXPECT* columns.
 
 [NOTE]
 ====
 For map entries, an entry key must be a `String` data type.
 ====
 
+To pass the parameter in the *EXPECT* column of Rule-based collection editor use the `actualValue` keyword whereas use the `?` keyword in DMN-based test scenario.
 
 .Procedure
 . Set the column type first (use a field whose type is a list or a map).
 . Double click a cell in the column to input a value.
-. In the collection editor popup, click *Add new item*.
-. Enter the required value and click the check icon image:dmn/dmn-datatype-constraints-tickmark.png[] to save each collection item that you add.
-. Click *Save*.
+. To create the list values for the data objects in the collection editor popup:
+.. Select *Create List*.
+.. Click *Add new item*.
+.. Enter the required value and click the check icon image:dmn/dmn-datatype-constraints-tickmark.png[] to save each collection item that you add.
+.. Click *Save*.
+.. To edit an item from the collection, click the pencil icon in the collection popup editor.
+.. Click *Save changes*.
+.. To delete an item from the collection, click the bin icon in the collection popup editor.
+
+. To define the list values for the data objects in the collection editor popup:
+.. Select *Define List*.
+.. Use the MVEL or FEEL expression to define a list values in the text field.
```

Comment:
... Use the MVEL or FEEL expression to define a list value in the text field. (remove plural with 'a' article)